ruby/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'tzinfo'
```

Comment:
We need tzinfo-data as well. Presumably bundler will then recreate the gemfile.lock appropriately...